definitions/release/install.yaml

```diff
@@ -535,7 +535,7 @@ spec:
     - jsonPath: .metadata.creationTimestamp
       name: Age
       type: date
-    name: v1alpha1
+    name: v1alpha2
```

Comment:
Let's not update release/install.yaml for now, we'll update it when there's a new release.

Reply:
Done